srcpkgs/walk/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'walk'
+pkgname=walk
+version=20190920
+revision=1
+_commit=70f7a8c104acf109ae810a9b34eb8e8b92b4d27d
+wrksrc="${pkgname}-${_commit}"
+build_style="gnu-makefile"
+short_desc="Two utilities collectively replacing find: walk and sor"
+maintainer="Orphaned <orphaned@voidlinux.org>"
```

Comment:
This is not the correct mail.

And given that this is a rather niche package, it should have a maintainer in order to be accepted.

Reply:
I took maintainership.